i18n/qqq.json

```diff
@@ -29,6 +29,7 @@
 	"header-textsize": "The header title of the text size page",
 	"menu-previous": "Label of the Previous article item. Shown in the Menu page.",
 	"menu-textsize": "Label of the Text size item list. Shown in the Menu menu.",
+	"more-information": "Label of the Text \"more information\". Used in Collapsed Table in Article page.",
```

Comment:
The first part is redundant since translator will see the English version on TWN. You can specify that selecting this element will open the table full screen in a new popup.